Sources/Sandbox.Game/Game/Entities/MyEntity.cs

```diff
@@ -100,6 +100,13 @@ public MyGameLogicComponent GameLogic
             set { Components.Add<MyGameLogicComponent>(value); }
```

Comment:
`Components.Add` does not happen for the elements of `m_gameLogicSet`. That can't be a good thing.

Looking at `Components.Add` I guess each logic component should be added with it's own type or the components will overwrite each other in the component-list.